include/bluetooth/conn.h

```diff
@@ -927,6 +930,18 @@ struct bt_conn_cb {
 				    struct bt_conn_le_data_len_info *info);
 #endif /* defined(CONFIG_BT_USER_DATA_LEN_UPDATE) */
 
+#if defined(CONFIG_BT_DF_CONNECTION_CTE_RX)
+	/**
+	 * @brief Callback for IQ samples report collected when sampling
```

Comment:
To be consistent with the callbacks above
```suggestion
	/** @brief Callback for IQ samples report collected when sampling
```